src/_helpers/__tests__/theme.test.js

```diff
@@ -0,0 +1,23 @@
+import * as themeHelpers from '../theme';
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('test "makePxValue" helper function', () => {
+  it('checks "makePxValue" is called with 1 argument', () => {
+    const spy = jest.spyOn(themeHelpers, 'makePxValue');
+    const value = themeHelpers.makePxValue(2);
+    expect(spy).toHaveBeenCalledTimes(1);
+    expect(spy).toHaveBeenLastCalledWith(2);
+    expect(value).toEqual('16px');
+  });
+
+  it('checks "makePxValue" is called with 1 argument(Array)', () => {
+    const spy = jest.spyOn(themeHelpers, 'makePxValue');
+    const value = themeHelpers.makePxValue([1, 2]);
+    expect(spy).toHaveBeenCalledTimes(1);
+    expect(spy).toHaveBeenLastCalledWith([1, 2]);
```

Comment:
What happens when you call it with array length > 4? It should break ideally. We need to handle that scenario